mmv1/third_party/terraform/website/docs/r/google_kms_key_ring_iam.html.markdown

```diff
@@ -51,7 +51,7 @@ resource "google_kms_key_ring" "keyring" {
 
 data "google_iam_policy" "admin" {
   binding {
-    role = "roles/editor"
+    role = "roles/cloudkms.admin"
```

Comment:
it seems there are two types of resources here. Does this apply to both `google_iam_policy` and `google_kms_key_ring_iam_*`?

Reply:
Oh you are right. My intention is only for google_kms_key_ring_iam_*. Let me update. Thanks